scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -168,36 +169,50 @@ sample_names <- c("SCPCL000479",
 sce_paths <- file.path(input_dir, 
                        glue::glue("{sample_names}.rds")
 )
+```
+
+```{r print sce paths, live = TRUE}
+# Print the sce_paths vector
 sce_paths
 ```
 
-We can now read these files in and create a list of four SCE objects:
+We can now read these files in and create a list of four SCE objects.
+Since `readr::read_rds()` can only operate on one input at a time (unlike `sqrt()` which can operate across a vector), we'll need to use `purrr::map()` to run in simultaneously on all input file paths.
```

Comment:
```suggestion
Since `readr::read_rds()` can only operate on one input (unlike `sqrt()` which can operate across a vector), we can use `purrr::map()` to read in all input file paths.
```
Technically it's still happening one by one and not simultaneously, so I did some small rewording to make that clear, we aren't running multiple things at once, we are reading all the files in, one by one, but doing it with one line of code.

Reply:
Maybe I'll add back in "in one command" instead of simultaneously (which makes sense!)